subsys/mgmt/mcumgr/lib/cmd/os_mgmt/src/os_mgmt.c

```diff
@@ -222,7 +207,7 @@ static int os_mgmt_taskstat_read(struct mgmt_ctxt *ctxt)
 		++thread_idx;
 	}
 
-	err |= cbor_encoder_close_container(&ctxt->encoder, &tasks_map);
+	err = err | cbor_encoder_close_container(&ctxt->encoder, &tasks_map);
```

Comment:
Is this unwanted line change?

Reply:
I have no explanation for the change, so yes unwanted. Will fix.

Reply:
err from the loop should propagate through to this, should it not?

Reply:
Yes, but the original line did that anyway.

Reply:
It does in this view but in the commit view, it shows this as the original:
```
 	err = cbor_encoder_close_container(&ctxt->encoder, &tasks_map);
```
Without the |, not sure why there's a difference in the views

Reply:
Done